data/ka'het/ka'het missions.txt

```diff
@@ -293,3 +293,409 @@ mission "Ringworld Debris: Quarg"
+			choice
+				`	"What is the next step?"`
+			`	"Well, first we need to install the scanners on your ship. Luckily, they are as simple to use as a normal outfit scanner, so you should not have much trouble once you reach the asteroid. After you have made extensive scans of the sites I indicate to you, reach me back here on Aventine so that I can analyze the data. If everything goes well, you should be back here in a couple weeks. I have already gotten a team ready to install the scanners; all that needs to be done now is for you to head back to <system>."`
+				accept
+	on enter "Patir"
+		dialog
+			`After a brief scan of the system, your ship's computers identify the asteroid that Dusk indicated you. Compared to the asteroids you are used to, this one seems orders of magnitude bigger.`
+
+event "asteroid on patir"
+	planet "Builder Settlement"
+		remove attributes "requires: ka'sei"
+
+mission "Ka'het: Patir Mystery 3"
+	landing
+	name "Return to <planet>"
+	description "Return to Dusk on <destination>."
+	source "Builder Settlement"
+	destination "Aventine"
+	cargo "short range scanner" 15
+	to offer
+		has "Ka'het: Patir Mystery 2: done"
+	on offer
+		conversation
+			`Your initial scans suggest that the asteroid is slowly rotating on itself. You set your ship on a polar orbit around it and activate the sensors that Dusk gave you, hopefully to find some answers to all the questions that Patir raised.`
+			`	You start your exploration from the dark side of the asteroid, and from the little you can see, the surface appears to be completely covered in regolith of little importance. But, soon after passing the terminator, you see an extremely reflective patch near the horizon, likely to be some sort of metal. As you get closer, it becomes clear that this is the storage of iridium that Dusk was talking about: in front of you, there is a huge reserve of the reflective metal you cannot see the end of, extending for kilometers in every direction inside a gigantic hole the Ka'sei dug in the terrain. The metal appears to be magnetic, as when you start passing over it some of the instruments aboard your spaceship go haywire. Luckily, Dusk's scanners remain unaffected, allowing you to continue the exploration.`
+			`	In a few hours, you are able to check every site you were told to scan, but most of the surface still remains an unknown.`
+			choice
+				`	(Continue your exploration of the asteroid.)`
+					goto explore
+				`	(Leave and return to Aventine.)`
+			`	As you are preparing to jump away, your eye is caught by an artificial structure near the edge of the iridium storage. The first thing you notice is an array of extremely high pylons put into two parallel lines; then, when you are able to get a better angle of view, you discover a large cupola towering all over the area. Although understanding its scale without a reference point is hard, you estimate it must be at least a kilometer in diameter.`
```

Comment:
```suggestion
			`	As you are preparing to jump away, an artificial structure near the edge of the iridium storage catches your eye. The first thing you notice is an array of tall pylons arranged into two parallel lines. When you are approach, you discover a large cupola towering all over the area. Although understanding its scale without a reference point is hard, you estimate it must be at least a kilometer in diameter.`
```

Reply:
This probably needs to be re-worked though. "Cupola" is probably not the best word here as it is not very descriptive. You might want to describe it in more detail, and also mention how it relates to the pylons. Where is it?

Reply:
I didn't bother describing it much as there is a scene of it next to this, so it would be rather redundant and less effective